tools/run_tests/dockerize/build_interop_image.sh

```diff
@@ -75,6 +75,22 @@ git branch -f jenkins-docker
 
 CONTAINER_NAME="build_${BASE_NAME}_$(uuidgen)"
 
+# build_interop.sh could be missing from old git tree and we will have issue unless we find the proper
+# build_interop.sh to use in such case.
+if ! [ -e $GRPC_ROOT/tools/dockerfile/interoptest/$BASE_NAME/build_interop.sh ]; then
```

Comment:
this is because grpc_interop_cxx_debian8/ is added to the HEAD while we are trying to find the build_interop.sh from an old tree.  Using build_interop.sh from HEAD to build old tree doesn't work as we have made in compatible changes.